bats.yaml

```diff
@@ -1,24 +1,22 @@
 package:
```

Comment:
writing some tests would be great here:

for example:
cat <<EOF > example.sh
#!/bin/bash

# Function to add two numbers
add_numbers() {
    echo $(( $1 + $2 ))
}

# Function to check if a file exists
file_exists() {
    if [[ -f "$1" ]]; then
        echo "true"
    else
        echo "false"
    fi
}

# Function to check if a string contains another string
string_contains() {
    if [[ "$1" == *"$2"* ]]; then
        echo "true"
    else
        echo "false"
    fi
}
EOF

cat <<EOF >
#!/usr/bin/env bats

# Load the script to test
load_file() {
    source "./example.sh"
}

setup() {
    # This runs before every test
    load_file
}

teardown() {
    # This runs after every test
    rm -f testfile.txt
}

@test "add_numbers should correctly add two numbers" {
    run add_numbers 3 5
    [ "$status" -eq 0 ]
    [ "$output" -eq 8 ]
}

@test "file_exists should return true for existing file" {
    touch testfile.txt
    run file_exists testfile.txt
    [ "$status" -eq 0 ]
    [ "$output" = "true" ]
}

@test "file_exists should return false for non-existing file" {
    run file_exists non_existing_file.txt
    [ "$status" -eq 0 ]
    [ "$output" = "false" ]
}

@test "string_contains should return true if substring is found" {
    run string_contains "hello world" "hello"
    [ "$status" -eq 0 ]
    [ "$output" = "true" ]
}

@test "string_contains should return false if substring is not found" {
    run string_contains "hello world" "bye"
    [ "$status" -eq 0 ]
    [ "$output" = "false" ]
}
EOF

Reply:
Sure thing!